src/components/credentials/__tests__/__snapshots__/credentialsEmptyState.test.js.snap

```diff
@@ -46,6 +46,7 @@ exports[`CredentialsEmptyState Component should render a basic component 1`] = `
             data-ouia-component-id="OUIA-Generated-Dropdown-1"
             data-ouia-component-type="PF4/Dropdown"
             data-ouia-safe="true"
+            togglevariant="primary"
```

Comment:
optional: try to determine if this is a pf issue or something we're doing in the dropdownSelect component

Reply:
seems to be an intermittent issue... specifically since we rolled button and select into the same component